Modules/System/Email/src/Email/Email.Codeunit.al

```diff
@@ -239,6 +239,17 @@ codeunit 8901 "Email"
         exit(EmailImpl.GetSentEmailsForRecord(TableId, SystemId));
     end;
 
+    ///<summary>
+    /// Gets the outbox emails related to a record.
+    ///</summary>
+    ///<param name="TableId">The table ID of the record.</param>
+    ///<param name="SystemId">The system ID of the record.</param>
+    ///<returns>The outbox emails related to a record.</returns>
+    procedure GetEmailOutboxForRecord(TableId: Integer; SystemId: Guid) ResultEmailOutbox: Record "Email Outbox" temporary;
```

Comment:
It would be better to get a Record Variant as a parameter, these values can be retrieved from the record variant and it would be a cleaner and simpler  interface.

Reply:
Same as in previous comment - two additional overloads for **GetEmailOutboxForRecord** and **GetSentEmailsForRecord** procedures were added to keep the same logic for both procedures. But I got an issue in test function. Will describe below.